### DISCUSSION TITLE
V0.1.1

### DISCUSSION THREAD

**Discussion by MFn233** [MFn233/PCL-Mainpage#41] [公告]
> Man, what can i say?

这个版本，我**修复了无法启动基岩版的问题**，现在你可以正常启动MC基岩版

（其实这是龙猫的问题，不过我找到了可行的方法。感谢 @erqianwu 提供了一点思路）

我修改了一些按钮上显示的文字，删去了不必要的“Minecraft”：

![image](https://github.com/MFn233/PCL-Mainpage/assets/103323756/d47537c0-1193-4bd6-a484-7e8a1e47b785)

我**删去了一些冗余的网页捷径**，留下我认为质量较高的。
网页捷径的“Minecraft国内”里增加了**MCNAV**，你现在可以在那里寻找更多网页捷径。

![image](https://github.com/MFn233/PCL-Mainpage/assets/103323756/ecd3f763-f922-4242-a41b-7820c032a20a)

![image](https://github.com/MFn233/PCL-Mainpage/assets/103323756/9c83279e-7841-4b35-8e70-2de181828c68)

真好，又能接着咕咕咕了~

<hr /><em>This discussion was created from the release <a href='https://github.com/MFn233/PCL-Mainpage/releases/tag/V0.1.1'>V0.1.1</a>.</em>

**Comment by MFn233** [comment]
哦对了，多提一嘴
要正常启动mc基岩版，你的电脑上必须有完整的system32和存放在里面的cmd.exe

: D

**Reply by SqrtTan** [reply]
谁家电脑没有啊（恼

**Reply by Ad-closeNN** [reply]
@MCSteve123 麻烦你访问一下mcnav，它貌似进不去了

**Reply by MFn233** [reply]
> @MCSteve123 麻烦你访问一下mcnav，它貌似进不去了

但是真的是正常的

**Reply by SqrtTan** [reply]
![image](https://github.com/user-attachments/assets/84423aeb-c891-4d52-826e-e90f6ede1553)
正常
~对不起今天才看到~

**Reply by Ad-closeNN** [reply]
emmm
~~为什么我今天能进去了~~
真奇怪前段时间还不能访问的
